src/components/PeopleList.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[];
+  setQuery: (query: string) => void;
+  setIsListShown: (argument: boolean) => void;
+  setSelectedPerson: (person: Person) => void;
+};
+
+export const PeopleList: React.FC<Props> = ({
+  people,
+  setQuery,
+  setIsListShown,
+  setSelectedPerson,
+}) => {
+  const handleItemClick = (person: Person) => {
+    setQuery(person.name);
+    setSelectedPerson(person);
+    setIsListShown(false);
+  };
+
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        {people.length
+          ? people.map((person) => (
+            <div
```

Comment:
when you move the mouse over names in the list it should be a cursor pointer (to show a user that it is clickable)

you can repalce `div ` by `a` tag in order to get that result or add by scss style